js/test/date/nodeunit/testdatefmt_ar_EG.js

```diff
@@ -0,0 +1,2138 @@
+        test.done();
+    },
+    
+    testDateFmtNativeSimpleTimeFull_ar_EG: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "ar-EG", useNative: false,  length: "full", type: "time"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "ar-EG",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute:45,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "‏1:45 م");
+        test.done();
+    },
+    
+    testDateFmtNativeDateTimeSimpleShort_ar_EG: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "ar-EG", useNative: false,  length: "short", type: "datetime"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "ar-EG",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute:45,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "‏29‏/9‏/2011 ‏1:45 م");
+        test.done();
+    },
+    
+    testDateFmtNativeDateTimeSimpleMedium_ar_EG: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "ar-EG", useNative: false,  length: "medium", type: "datetime"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "ar-EG",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute:45,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "‏29‏/09‏/2011 ‏1:45 م");
+        test.done();
+    },
+    
+    testDateFmtNativeDateTimeSimpleLong_ar_EG: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "ar-EG", useNative: false,  length: "long", type: "datetime"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "ar-EG",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute:45,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "‏29 سبتمبر، 2011 ‏1:45 م");
+        test.done();
+    },
+    
+    testDateFmtNativeDateTimeSimpleFull_ar_EG: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "ar-EG", useNative: false,  length: "full", type: "datetime"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "ar-EG",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute:45,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "‏29 سبتمبر، 2011 ‏1:45 م");
+        test.done();
+    }
+    
```

Comment:
How about a few tests with the islamic calendar?

Reply:
oh, Always I miss release note again. I'll add release note and more testcaes.
fyi. According to CLDR (cldr-core/supplemental, calendarPreferenceData.json), a calendar preference are as follows. A default is `gregorian`. 
``` "EG": "gregorian coptic islamic islamic-civil islamic-tbla"  ```